tests/integration/wait-until-test.js

```diff
@@ -109,5 +110,21 @@ test('The promise resolved with the return value of the callback', async functio
   assert.expect(1);
   this.render(hbs`<span id="wait-until-thing"></span>`);
   let element = await waitUntil(() => document.querySelector('#wait-until-thing'));
-  assert.equal(element.id, 'wait-until-thing', 'waitUntil resoleved with the DOM element');
+  assert.equal(element.id, 'wait-until-thing', 'waitUntil resolved with the DOM element');
 });
+
+test('When callback immediately returns truthy, promise is resolved and callback is not called again', async function(assert) {
+  let count = 0;
+  function cb() {
+    count++;
+    return true;
+  }
+  let initialTime = +new Date();
```

Comment:
super minor nit-pick, would you mind swapping this for `Date.now()` instead?  `+new Date()` _slightly_ hurts my head...